configs/ext/README.md

```diff
@@ -0,0 +1,20 @@
+Summary
+-------
+
+All files in this directory are distributed under the normal Mbed TLS dual Apache 2.0 or GPLv2-or-later
+license, and are copyright The Mbed TLS Contributors.
+
+Background
+-----------
+
+The two files crypto_config_profile_medium.h and tfm_mbedcrypto_config_profile_medium.h
+are taken verbatim from the TF-M source code here:
+
+https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git/tree/lib/ext/mbedcrypto/mbedcrypto_config
+
+In TF-M, they are distributed under a 3-Clause BSD license, as noted at the top of the files.
+
+In Mbed TLS, with permission from the TF-M project, they are distributed under a dual [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) OR [GPL-2.0-or-later](https://spdx.org/licenses/GPL-2.0-or-later.html) license, with copyright assigned to The Mbed TLS Contributors.
+
+We only retain the note at the top of the files because we are taking the files un-modified, for ease of
```

Comment:
`unmodified` doesn't need a hyphen, but actually `verbatim` might be a better word, as used in the PR title